readme.md

```diff
@@ -16,51 +16,59 @@ By convention, when the artifact is a library, the files are called
 ## Examples
 
 - `covariate-shift` Determine if there is a shift in data distribution
-  between two datasets.
-- `model-or-ensemble` Decide whether to use models or ensembles for
-  predictions, given an input source.
+  between two datasets.  
+- `model-or-ensemble` Decide whether to use
+  models or ensembles for predictions, given an input source.  
```

Comment:
hmm... you seem to have changed all the indentation in this file (with trailing whitespace too), for no good reason?